model/enduser/deprecated/registry-deprecated.yaml

```diff
@@ -2,23 +2,27 @@ groups:
   - id: registry.enduser.deprecated
     type: attribute_group
     display_name: Deprecated End User Attributes
-    brief: Describes deprecated enduser attributes. Complete enduser namespace has been deprecated
+    brief: "Describes deprecated end user attributes."
     attributes:
       - id: enduser.id
         type: string
+        brief: 'Deprecated, use `enduser.pseudo.id` instead.'
```

Comment:
I believe `enuser.id` should stay as it is. Even if we call it `enduser.authentication.id`, the old `enduser.id` matches authenticated one.

Reply:
the main reason we came up with `enduser.pseudo.id` or a list of other naming options because we don't want customers to put authenticated user id to `enduser.id`. authenticated one is using `enduser.authentication.id`.

Reply:
the ask is to keep `enuser.id` instead of adding `enuser.authentication.id`. This PR would add `enuser.pseudo.id` and un-deprecate `enuser.id`

Reply:
that wasn't what we discussed though. what about the sub-namespace authentication? i thought that is what we want to do so that any other attributes associated with `authentication` can be added later. 

if you want to change it now, that will be different. we would need to re-discuss it. i thought the consensus is to have authentication sub namespace.

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/1104#issuecomment-2259426406 it was from you @lmolkova

Reply:
it's been under the assumption that we use `user.id` for pseudo/tracking/anonymous id - as you can see we've reconsidered it after that.

> if you want to change it now, that will be different. we would need to re-discuss it.

It seems at least me and @trask are on the same page to use `enduser.id` instead of `enduser.authentication.id`. I believe I saw some support from @trisch-me and @jsuereth on it too. 

Please make the change and if anyone has objections they will be able to comment and provide reasoning.